Covid Track/Covid Track/ViewModels/Visit Record/VisitRecordViewModel.swift

```diff
@@ -0,0 +1,81 @@
+//
+//  VisitRecordViewModel.swift
+//  Covid Track
+//
+//  Created by Samip shah on 28/7/20.
+//  Copyright © 2020 Samip shah. All rights reserved.
+//
+
+import Foundation
+
+class VisitRecordError:Error {
+  var localizedDescription: String
+  
```

Comment:
why localizedDescription is publically assinable?

Reply:
Access modifier has been changed to private set.